system-parachains/collectives/collectives-polkadot/src/impls.rs

```diff
@@ -159,6 +164,51 @@ impl PrivilegeCmp<OriginCaller> for EqualOrGreatestRootCmp {
 	}
 }
 
+/// Contains a system-level sibling parachain.
+pub struct IsSiblingSystemParachain<ParaId>(PhantomData<ParaId>);
+impl<ParaId: IsSystem + From<u32>> Contains<Location> for IsSiblingSystemParachain<ParaId> {
+	fn contains(l: &Location) -> bool {
+		matches!(
+			l.interior().as_slice(),
+			[Junction::Parachain(id)]
+				if ParaId::from(*id).is_system() && l.parent_count() == 1,
```

Comment:
This would also return `true` when a parachain puts its own `ParaId`? Dont know if its a problem, but a parachain being its own sibling seems weird to me.

Reply:
Good point